packages/core/src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -188,7 +245,22 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
             style={{ width: inputWidth }}
             role="input"
           />
-        )}
+        );
+      }
+    }
+
+    return (
+      <div
+        ref={mergedRef}
+        id={id}
+        aria-label={ariaLabel}
+        data-testid={dataTestId}
+        className={cx(styles.editableTypography, className)}
+        role={isEditing ? null : "button"}
```

Comment:
It will not be shown when not editing, plus role null is bad for a11y

Reply:
that part existed already, didn't want to change as i don't know what it breaks, but lets discuss and i'll change accordingly